### DISCUSSION TITLE
Disable main split button in dropdown

### DISCUSSION THREAD

**Discussion by lucianobosco** [bootstrap-vue/bootstrap-vue#6790] [Q&A] [answered]
I'm trying to disable the main split button from a dropdown but keep the dropdown group enabled

This is the most basic sample code from the [docs][1]:
```
<div>
  <b-dropdown split text="Split Dropdown" class="m-2">
    <b-dropdown-item href="#">Action</b-dropdown-item>
    <b-dropdown-item href="#">Another action</b-dropdown-item>
    <b-dropdown-item href="#">Something else here...</b-dropdown-item>
  </b-dropdown>
</div>
```
By setting `:disabled="true"` in `<b-dropdown>` disables the whole button and I'm not able to expand the other options.

Using the slot `button-content` ignores the `disabled` property which makes sense because I'm overwriting the content and not the button itself.
```
<template slot="button-content" :disabled="true">Split Dropdown</template>
```
Is there a way to achieve this?

  [1]: https://bootstrap-vue.org/docs/components/dropdown#split-button-support

**Comment by Hiws** [comment, marked as answer]
All the `split` prop does AFAIK is adding a `<b-button-group>` around the two buttons.

You can instead do this yourself to get the control you need.
```html
<b-button-group>
  <b-button disabled>Split dropdown</b-button>
  <b-dropdown>
    <b-dropdown-item href="#">Action</b-dropdown-item>
    <b-dropdown-item href="#">Another action</b-dropdown-item>
    <b-dropdown-item href="#">Something else here...</b-dropdown-item>
  </b-dropdown>
</b-button-group>
```

**Comment by lucianobosco** [comment]
Thanks! That solved my issue
